proto/multipoolermanagerservice.proto

```diff
@@ -79,6 +79,11 @@ service MultiPoolerManager {
   // The following methods are used by MultiOrch to
   // perform HA operations (reparents and failovers)
 
+  // GetDurabilityPolicy retrieves the active durability policy from the local database
+  // Used by MultiOrch to query quorum rules via gRPC instead of direct database connection
+  rpc GetDurabilityPolicy(multipoolermanagerdata.GetDurabilityPolicyRequest)
```

Comment:
Does this belong here, or should it be part of the consensus service? I couldn't make up my mind about this.